src/cli.ts

```diff
@@ -51,26 +51,27 @@ export function printUsage() {
    zx [options] <script>
 
  ${chalk.bold('Options')}
-   --quiet              suppress any outputs
-   --verbose            enable verbose mode
-   --shell=<path>       custom shell binary
-   --prefix=<command>   prefix all commands
-   --postfix=<command>  postfix all commands
-   --cwd=<path>         set current directory
-   --eval=<js>, -e      evaluate script
-   --ext=<.mjs>         default extension
-   --install, -i        install dependencies
-   --version, -v        print current zx version
-   --help, -h           print help
-   --repl               start repl
-   --experimental       enables experimental features (deprecated)
+   --quiet                suppress any outputs
```

Comment:
Now we can keep the formatting here for git blame.

Reply:
Thanks to your suggestion. Revert formatting